### PR TITLE
Add documentation about vizualizing the results

### PR DESCRIPTION
**Pull request by padovan**
Basic page to list our current was to interact with tests results.

### CODE REVIEW

File: kernelci.org/content/en/kernel-community/view-results.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+title: "Evaluating Test Results"
+date: 2024-09-06
+description: "Describe ways to interact with the KernelCI test results"
+---
+
+The Kernel Community have a few options to interact with the test results available in the KernelCI. Remember this data include results from several CI systems.
```
Comment (JenySadadia) [inline kernelci.org/content/en/kernel-community/view-results.md:7]:
```suggestion
The Kernel Community has a few options to interact with the test results available in the KernelCI. Remember this data includes results from several CI systems.
```

File: kernelci.org/content/en/kernel-community/view-results.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+* [**Web Dashboard**](https://dashboard.kernelci.org/)(WIP): Our new Web Dashboard aims at providing an easy way for the community to look at the tests results. It is still under development and we open to your [feedback and feature requests](https://github.com/kernelci/dashboard).
```
Comment (JenySadadia) [inline kernelci.org/content/en/kernel-community/view-results.md:9]:
```suggestion
* [**Web Dashboard**](https://dashboard.kernelci.org/)(WIP): Our new Web Dashboard aims at providing an easy way for the community to look at the test results. It is still under development and we are open to your [feedback and feature requests](https://github.com/kernelci/dashboard).
```

File: kernelci.org/content/en/kernel-community/view-results.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+* [**Grafana**](https://grafana.kernelci.org/): Our Grafana instance allow users to create specific dashboard with tailored queries and boards. Those who want to dive deeper in their data can engage with our Grafana instance.
```
Comment (JenySadadia) [inline kernelci.org/content/en/kernel-community/view-results.md:10]:
```suggestion
* [**Grafana**](https://grafana.kernelci.org/): Our Grafana instance allow users to create specific dashboard with tailored queries and boards. Those who want to dive deeper into their data can engage with our Grafana instance.
```

File: kernelci.org/content/en/kernel-community/view-results.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+* **Email reports**: [KCIDB](../../kcidb) allows users to create tailored email reports through [templates](https://github.com/kernelci/kcidb/tree/main/kcidb/templates).
```
Comment (JenySadadia) [inline kernelci.org/content/en/kernel-community/view-results.md:11]: Do you think it is worth mentioning the mailing list (staging results) we are sending email reports to?

Comment (padovan) [reply kernelci.org/content/en/kernel-community/view-results.md]: Yes. Just added a mention to that.

File: kernelci.org/content/en/kernel-community/view-results.md (hunk 3 above)
Comment (padovan) [reply kernelci.org/content/en/kernel-community/view-results.md]: Added these manually.

File: kernelci.org/content/en/kernel-community/view-results.md (hunk 2 above)
Comment (padovan) [reply kernelci.org/content/en/kernel-community/view-results.md]: Added these manually.

**Review by patersonc — APPROVED** [review]
LGTM

Thanks

**Pull request merged**